### DISCUSSION TITLE
Support source code managed by SVN

### DISCUSSION THREAD

**Discussion by louislau86** [microsoft/vcpkg#21891] [Ideas]
There are many software teams managed their source code in private SVN server, is there any way to use vcpkg manage the dependency of these modules in SVN?

**Comment by Osyotr** [comment]
Since vcpkg is just a pretty wrapper around cmake's `execute_process`, you can write your own `vcpkg_from_svn` function!
Take this as an example: https://github.com/microsoft/vcpkg/blob/master/scripts/cmake/vcpkg_from_git.cmake

**Comment by JackBoosY** [comment]
Good idea!

**Comment by motazmuhammad** [comment]
@JackBoosY did you manage to write the vcpkg_from_svn file?
